HTML/launches.html

```diff
@@ -263,6 +263,52 @@ <h1 class="trip-title">TRIP NAME</h1>
         
               </div>
       </div>
+
+      <div class="sponsors">
+        <div class="sponsor">
+          <img src="../images/sponsor.png" alt="sponsor1" class="sponsor-img" />
+        </div>
+        <div class="sponsor">
+          <img src="../images/sponsor.png" alt="sponsor2" class="sponsor-img" />
+        </div>
+        <div class="sponsor">
+          <img src="../images/sponsor.png" alt="sponsor3" class="sponsor-img" />
+        </div>
+        <div class="sponsor">
+          <img src="../images/sponsor.png" alt="sponsor4" class="sponsor-img" />
+        </div>
+        <div class="sponsor">
+          <img src="../images/sponsor.png" alt="sponsor5" class="sponsor-img" />
+        </div>
+        <div class="sponsor">
+          <img src="../images/sponsor.png" alt="sponsor6" class="sponsor-img" />
+        </div>
+      </div>
+
+      <footer class="footer">
+        <div class="footer-text">Sanajib Team © 2022</div>
+        <div class="flex-footer">
+          <img
+            src="../images/footer-image1.png"
+            alt="footer-image1"
+            class="footer-image1"
+          />
+          <ul class="footer-list">
+            <a class="footer-link" href="#"
+              ><i class="fab fa-facebook-f"></i
+            ></a>
+            <a class="footer-link" href="#"><i class="fab fa-instagram"></i></a>
+            <a class="footer-link" href="#"><i class="fab fa-twitter"></i></a>
+            <a class="footer-link" href="#"><i class="fab fa-linkedin"></i> </a>
+            <a class="footer-link" href="#"><i class="fab fa-whatsapp"></i> </a>
+          </ul>
+          <img
+            src="../images/footer-image2.png"
+            alt="footer-image2"
+            class="footer-image2"
+          />
```

Comment:
reduce the `font-size` of the social media icons, to make some space between the icons

<img width="595" alt="Screen Shot 2022-03-23 at 9 40 34 PM" src="https://user-images.githubusercontent.com/47992412/159782650-d3782f53-6928-4c98-8f5a-04610fc37591.png">

after updating the font-size 

<img width="595" alt="Screen Shot 2022-03-23 at 9 45 20 PM" src="https://user-images.githubusercontent.com/47992412/159783240-a674fbda-6889-459e-9751-982dcca726dd.png">

Reply:
thanks, done